test/cpp-arguments.test.js

```diff
@@ -21,19 +21,23 @@ describe('cpp-arguments', function () {
 	
 	describe('`assertion` function', function () {
 		it('should create assertions for base assertion types', function () {
+			function TestObject() {}
+			
 			var baseAssertions = {
 				'bool': false,
 				'string': "abc",
 				'int': 123,
 				'double': 0.5,
-				'object': { test: { something: "foo" } }
+				'object': new TestObject(),
+				'plainObject': { test: { something: "foo" } }
```

Comment:
Would it be fair to initialize this `plainObject` with the fields `TestObject` initializes itself with?